qa/rpc-tests/p2p_node_bloom.py

```diff
@@ -78,8 +78,8 @@ def run_test(self):
         assert_equal(1, versions.count(MY_VERSION))
 
```

Comment:
Add a check that sending ``filterclear`` does not cause the NoBF mininode to be dropped (to test 01c22a65899f721cd7355f711e5b5f3c97c10e0b).